core/common/src/thrift/block_master.thrift

```diff
@@ -56,7 +46,46 @@ struct GetUsedBytesTResponse {
   1: i64 bytes
 }
 
+enum WorkerRange {
+  ALL
+  LIVE
+  LOST
+  SPECIFIED
+}
+
+enum WorkerInfoField {
+  ADDRESS
+  CAPACITY_BYTES
+  CAPACITY_BYTES_ON_TIERS
+  ID
+  LAST_CONTACT_SEC
+  START_TIME_MS
+  STATE
+  USED_BYTES
+  USED_BYTES_ON_TIERS
+}
+
+struct WorkerInfo {
+  1: common.WorkerNetAddress address
```

Comment:
these integer (id) fields need to be the same as before if we want to keep backward compatibility.
e.g., `4: i64 id` will not work , change it to `1: i64 id`, that's what @aaudiber means by only "adding" new fields, @LuQQiu

Reply:
Yes! I tried using different version server and client. If the existing sequence does not match, then we will not get any workerInfo. 
Changed back and add new fields at the end. Thanks!
```
struct WorkerInfo {
  1: i64 id
  2: common.WorkerNetAddress address
  3: i32 lastContactSec
  4: string state
  5: i64 capacityBytes
  6: i64 usedBytes
  7: i64 startTimeMs
  8: map<string, i64> capacityBytesOnTiers;
  9: map<string, i64> usedBytesOnTiers;
}
```